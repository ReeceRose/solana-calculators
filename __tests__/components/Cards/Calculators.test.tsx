import { render } from '@testing-library/react';

import CalculatorsCard from '../../../components/Cards/Calculators';

test('Calculators card component matches snapshot', () => {
  const { asFragment } = render(<CalculatorsCard />);
  expect(asFragment()).toMatchSnapshot();
});
