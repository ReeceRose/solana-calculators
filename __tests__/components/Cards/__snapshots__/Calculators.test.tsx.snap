// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calculators card component matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="flex-auto px-4 lg:px-10"
  >
    <hr
      class="my-6 border-gray-50 border-b-1"
    />
    <h6
      class="text-sm font-bold uppercase text-gray-50"
    >
      Calculators
    </h6>
  </div>
  <div
    class="flex-auto px-4 mb-6 lg:px-10"
  >
    <hr
      class="my-6 border-gray-50 border-b-1"
    />
    <div
      class="w-full px-4 "
    >
      <p
        class="block mb-2 text-xs font-bold uppercase text-gray-50"
      >
        <a
          class="text-solana-green hover:text-solana-green-hover"
          href=""
        >
          Rent Calculator - 
          <span
            class="text-gray-50"
          >
             Calculate rent cost
          </span>
        </a>
      </p>
    </div>
  </div>
</DocumentFragment>
`;
